SPECS-EXTENDED/gspell/gspell.spec

```diff
@@ -4,16 +4,17 @@ Distribution:   Azure Linux
 %global gtk3_version 3.20
 
 Name:           gspell
-Version:        1.8.4
-Release:        2%{?dist}
+Version:        1.14.0
+Release:        1%{?dist}
 Summary:        Spell-checking library for GTK+
 
-License:        LGPLv2+
-URL:            https://wiki.gnome.org/Projects/gspell
-Source0:        https://download.gnome.org/sources/%{name}/1.8/%{name}-%{version}.tar.xz
+License:        LGPLv2.1+
+URL:            https://gitlab.gnome.org/GNOME/gspell
+Source0:	https://gitlab.gnome.org/GNOME/gspell/-/archive/%{version}/%{name}-%{version}.tar.gz
```

Comment:
Please replace tab with spaces

Reply:
My editor was not configured to show non-printing characters, I'll fix that now.